modules/ROOT/pages/external-identity.adoc

```diff
@@ -13,7 +13,7 @@ The following diagram shows the SAML identity management process:
 
 image::external-identity-decbd.png[A diagram showing the SAML identity management flow]
 
-Before configuring OpenID Connect or SAML 2.0, select an OpenID Connect-compliant provider, such as PingFederate, OpenAM, or Okta. If you are not using OpenID Connect, select any SAML 2.0-compliant provider.
+Before configuring OpenID Connect or SAML 2.0, select an OpenID Connect-compliant provider, such OpenAM or Okta. If you are not using OpenID Connect, select any SAML 2.0-compliant provider.
```

Comment:
Thanks @valkyrie69 for spotting this, but this is Identity Management for logging in users and I didn't change its functions. Let's keep text as is?